src/gui/auxiliary/field-fwd.hpp

```diff
@@ -23,19 +23,19 @@
 namespace gui2
 {
 
-class twidget;
-class twindow;
+class widget;
+class window;
 
-class tfield_;
-class tfield_bool;
-class tfield_label;
-class tfield_text;
+class field_base;
```

Comment:
what kind of field are these? - better make that clear here if you're changing the types.